portal/templates/redesign/about_new.html

```diff
@@ -0,0 +1,138 @@
+{% extends 'redesign/base_new.html' %}
+{% load staticfiles %}
+{% load app_tags %}
+
+{% block css %}
+{{ block.super }}
+{% endblock css %}
+
+{% block scripts %}
+{{ block.super }}
+{% endblock scripts %}
+
+{% block pageID %}id='homeWrapper'{% endblock %}
+
+{% block content %}
+{{ block.super }}
+
+<div class="banner banner--teacher row">
+    <div class="col-sm-5 col-sm-offset-2">
+        <h1 class="banner__text--primary">About Code for Life</h1>
+        <h4>Code For Life gives everyone the ability to shape technology&rsquo;s future</h4>
+    </div>
+    <div class="col-sm-5">
+        <div class="hexagon"><div class="hexagon-in1"><div class="hexagon-in2 hexagon-in2--about"></div></div></div>
```

Comment:
Is your "hexagon" class referring to something hexagonal?  If so, you need to think about why it's a hexagon, as at some point you may want it to be some other shape and your hexagon class isn't going to make sense